src/main/java/com/iexec/common/sdk/marketplace/Marketplace.java

```diff
@@ -33,9 +33,13 @@
 
     public Optional<AppMarketOrder> getOneAppOrder(String appAddress);
 
+    public Optional<AppMarketOrder> getOneRequesterRestrictedAppOrder(String appAddress, String requesterAddress);
```

Comment:
What about `getOneAppOrder(String appAddress, String requesterAddress)`?

Reply:
Yeah, I thought about that, but if we want to add `getAppOrder(String appAddress, String appOwner)` it wouldn't be possible.

Reply:
:+1: